### ISSUE TITLE
Python scripts shebang

### ISSUE THREAD

**Issue by MichaIng** [HTPC-Manager/HTPC-Manager#32]
While this is aimed to be run with Python 3, all shebangs  `#!/usr/bin/env python` would usually call Python 2. At least on all distros I know, the `python` command calls Python 2 while `python3` is required to call Python 3, and additional packages or manual symlinks/aliases are required to make `python` calling Python 3. But, `#!/usr/bin/env python` is still the pretty default shebang even on pure Python 3 projects, which causes issues here and there when a script is wanted as executable in PATH, e.g. `youtube-dl` is such an example which is Python 3 compatible, but many users unintentionally execute it with Python 2 or get an error.

I'm not sure whether it is good to keep the convention or if it makes sense to switch to `#!/usr/bin/env python3`, so that `/path/to/Htpc.py` can be used to call HA, instead of `python3 /path/to/Htpc.py`, and to make pretty clear that it is a Python 3 project within each script?

**Comment by gmiranda** [comment]
First of all, yes it does make sense to replace with python3 in the shebangs. In my current system there's no python2 and env python returns null.

That said, I'm using a virtual environment for HTPCManager, and env python returns the python3 interpreter when using the venv.

**Comment by MichaIng** [comment]
Yes in venv and pyenv it should work indeed. I guess it has no practical effect currently, but is more some kind of statement 😄.
